mattermost/mattermost-plugin-nps
# [MM-18640] Update mattermost-redux

#### Summary

This repo is already manually using the CSRF token, but I figure all of the plugins should use it the same way (through the `mattermost-redux` repo).

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-18640

#### QA Steps

- Set `ServiceSettings.ExperimentalStrictCSRFEnforcement` to true in the server's config.json

- When refreshing the page, there should be a network call to `/plugins/com.mattermost.nps/api/v1/connected`. It should return a 200.
